webapp/portlet/src/main/webapp/WEB-INF/jsp/bannerLogo.jsp

```diff
@@ -35,13 +36,20 @@
     logoTitle = space.getDisplayName();
     String permanentSpaceName = space.getGroupId().split("/")[2];
     portalPath = "/portal/g/:spaces:" + permanentSpaceName + "/" + space.getPrettyName();
-    imageClass="spaceAvatar";
-    titleClass = "space";
+    membersNumber = space.getMembers().length;
```

Comment:
This will be a performance killer. Could you find another way to do it ?

Reply:
the space.getMmebers is an array of String (usernames of memebers) , when call that method no rest call will be invoked and no computed functions will be called. all the model (space data) already loaded in the cache of the server, and  i just access and use these already loaded data.